ext/bg/js/search.js

```diff
@@ -202,6 +198,17 @@ class DisplaySearch extends Display {
         }
     }
 
+    onCopy() {
+        // ignore copy from search page
+        this.clipboardMonitor.setPreviousText(document.getSelection().toString().trim());
+    }
```

Comment:
I might add a similar check to the global clipboard monitor that can be used to toggle the feature on/off within the browser without affecting other software. It could be less reliable because it takes some time to serialize API commands and meanwhile the `setTimeout` can finish.

Reply:
I won't do this for now because of its complexity and small benefit (has to run in every frame)